src/drone_steering/arm_test.py

```diff
@@ -0,0 +1,87 @@
+# latanie irl
+
+from dronekit import connect, VehicleMode, LocationGlobalRelative, APIException
+import time
+import socket
+# import exceptions
+import math
+import argparse
+
+## ==== connection ==== ##
+
+def connectMyCopter():
+    parser = argparse.ArgumentParser(description='commands')
+    parser.add_argument('--connect', default='127.0.0.1:14550')
+    args = parser.parse_args()
+
+    connection_string = args.connect
+    baud_rate = 57600
+
+    vehicle = connect(connection_string, baud=baud_rate, wait_ready=False) #doesnt work with wait_ready=True
+    return vehicle
+
+def arm():
+    while vehicle.is_armable==False:
+        print("Waiting for vehicle to become armable...")
+        time.sleep(1)
+    print("Vehicle is now armable")
+    print("")
+
+    vehicle.armed=True # nie wywala bledow dzieki temu potem
+    while vehicle.armed==False:
+        print("Waiting for drone to become armed...")
+        time.sleep(1)
+
+    print("Vehicle is now armed.")
+    print("props are spinning!")
+
+    return None
+
+def get_distance_metres(aLocation1, aLocation2):
```

Comment:
built-in dronekit and ardupilot functions need to be linked in a smart way, not just pasted in